### DISCUSSION TITLE
Issue launching tasks

### DISCUSSION THREAD

**Discussion by Joan-Goma** [hibiken/asynq#443] [Q&A] [answered]
Hi!
I have been using asynq for a long time working correctly, but now, when I try to launch tasks, the "worker" logs tan error, and keep the task as active.

LOG:
ERROR: Dequeue error: UNKNOWN: redis eval error: ERR Error running script (call to f_740642ef104f90e8b06e352a28a9df688c72acf6): @user_script:14: user_script:14: attempt to perform arithmetic on local 'timeout' (a nil value)

**Comment by hibiken** [comment, marked as answer]
Hi @Joan-Goma thank you for raising this issue!
My guess is that you have version mismatch between the client and server.
[Version 0.22](https://github.com/hibiken/asynq/releases/tag/v0.22.0) changed how Asynq handles worker crashes and as part of that we no longer writes to the `timeout` and `deadline` fields under the task key.

Can you check if the program enqueueing the tasks is using the asynq version  0.22 or above. And the server program (the workers) are using asynq version 0.21 or below?

**Comment by Joan-Goma** [comment]
Hi @hibiken,
Yes this was the issue.
Thanks!
